server/modules/analysis/api/chainApi.js

```diff
@@ -162,12 +165,25 @@ export const init = (app) => {
         const { surveyId, surveyCycleKey, chainUuid } = Request.getParams(req)
         const serverUrl = Request.getServerUrl(req)
 
-        await AnalysisService.generateScript({ surveyId, cycle: surveyCycleKey, chainUuid, serverUrl })
-
-        Response.sendOk(res)
+        const rChain = await AnalysisService.generateScript({ surveyId, cycle: surveyCycleKey, chainUuid, serverUrl })
+        const folderToken = uuidv4()
+        await FileUtils.copyDir({ source: rChain._dir, destination: `/tmp/${surveyId}_${chainUuid}_${folderToken}/` })
+        res.json({ folderToken, serverUrl })
       } catch (error) {
         next(error)
       }
     }
   )
+
+  // === Download R SCRIPTS
+  app.get('/download/survey/:surveyId/processing-chain/:chainUuid/script', (req, res, next) => {
```

Comment:
for convention I would set his as `/survey/:surveyId/processing-chain/:chainUuid/scrip/download`

Reply:
Done!

Reply:
I did public all of the routes with download, should we add some subfix or something?

Reply:
/public ?

Reply:
done